doc/tutorial/fraud-analysis.md

```diff
@@ -0,0 +1,45 @@
+# Customize Data Analysis on SQLFlow Tutorial
+
+Data analysis can help us understand what is in the dataset and the
+characteristics of the data.
+
+Data binning is a common used data analysis way. It can group continous values
+into a small number of discretized bins. We will get the distribution of the
+data from the binning result.
+
+We can use SQLFlow TO RUN statement to execute the runnable which is released in the
+form of Docker image. SQLFlow provides some premade runnables in
+sqlflow/runnable including the binning runnable. Please use the following SQL
+statement to do the data binning.
+
+```SQL
+SELECT * FROM creditcard.creditcard
+TO RUN sqlflow/runnable:v0.0.1
+CMD "binning.py",
+    "--dbname=creditcard",
+    "--columns=time,v1,v2,v3,v4,v5,v6,v7,v8,v9,v10,v11,v12,v13,v14,v15,v16,v17,v18,v19,v20,v21,v22,v23,v24,v25,v26,v27,v28,amount",
```

Comment:
Just curious, there are so many columns, which one is used for binning? Or, would please explain a little bit of the logic behind the SQL statement?

Reply:
All the columns will be binned to 10 buckets.